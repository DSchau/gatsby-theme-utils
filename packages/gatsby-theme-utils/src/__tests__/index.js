const utils = require('..');

test('it has expected exports', () => {
  expect(Object.keys(utils)).toMatchSnapshot();
});
