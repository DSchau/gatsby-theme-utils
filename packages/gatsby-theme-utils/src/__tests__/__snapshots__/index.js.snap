// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it has expected exports 1`] = `
Array [
  "addToWebpackConfig",
]
`;
